components/zenFunctions/client/fn_zen_createJammer.sqf

```diff
@@ -0,0 +1,24 @@
+#include "macros.hpp"
+
+params ["_object"];
+
+if (isNull _object) then
+{
+	systemChat "No object selected.";
+}
+else
+{
+	["Create Jammer", 
+	[
+		["COMBO", "Jammer Type", [[0, 1, 2, 3, 4, 5, 6], ["All Types", "Radio", "Phones (ACE)", "Detonators (Clackers)", "Drones", "GPS", "Radar"], 0]],
```

Comment:
A similar suggestion to the one from wound-units-randomly, if there's some config or variables we can pull these values from instead of duplicating them here, then we insulate ourselves against change to a certain extent.  However, it's totally fine if there isn't and I'm not demanding that you action this point either.